app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
 @app.route('/recommend/popular')
 def popular_movies():
     data = recommend_popular_movies()
     return data
 
 
```

Comment:
This line is outside of any function or route handler, so it will cause a NameError.